pkg/cmd/roachtest/roachtestutil/mixedversion/runner.go

```diff
@@ -386,55 +364,22 @@ func (tr *testRunner) stepError(
 func (tr *testRunner) testFailure(
 	ctx context.Context, err error, l *logger.Logger, testContext *Context,
 ) error {
-	detailsForService := func(service *serviceRuntime) *serviceFailureDetails {
-		return &serviceFailureDetails{
-			descriptor:            service.descriptor,
-			binaryVersions:        loadAtomicVersions(service.binaryVersions),
-			clusterVersionsBefore: loadAtomicVersions(service.clusterVersions),
-			clusterVersionsAfter:  loadAtomicVersions(service.clusterVersions),
-		}
-	}
-
-	var systemDetails *serviceFailureDetails
-	var tenantDetails *serviceFailureDetails
-	for _, service := range tr.allServices() {
-		if service.descriptor.Name == install.SystemInterfaceName {
-			systemDetails = detailsForService(service)
-		} else {
-			tenantDetails = detailsForService(service)
-		}
-	}
-
-	currentClusterVersions := func(service *serviceRuntime) []roachpb.Version {
-		if tr.connCacheInitialized(service) {
-			if err := tr.refreshClusterVersions(ctx, service); err == nil {
-				return loadAtomicVersions(service.clusterVersions)
-			} else {
-				tr.logger.Printf(
-					"failed to fetch cluster versions for service %s after failure: %s",
-					service.descriptor.Name, err,
-				)
-			}
-		}
-
-		return loadAtomicVersions(service.clusterVersions)
-	}
-
-	systemDetails.clusterVersionsAfter = currentClusterVersions(tr.systemService)
-	if tenantDetails != nil {
-		tenantDetails.clusterVersionsAfter = currentClusterVersions(tr.tenantService)
+	lines := []string{
+		"test failed:",
+		fmt.Sprintf("test random seed: %d\n", tr.plan.seed),
```

Comment:
Not related to this PR, but should we explicitly mention that the seed can be set through `COCKROACH_RANDOM_SEED`? Since we saw someone ask about the diff between this and `global-seed`

Reply:
Done.